### PR TITLE
Pings frontend

### PR DESCRIPTION
**Pull request by Vexrax**
image

Fastest frontend dev in risen btw

### CODE REVIEW

File: front-end/src/components/player-page/stats/ping-overview.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+import {StatGenerators} from "../../../common/constants";
+import {BaseStatGenerator} from "../../../common/stats-generators/BaseStatsGenerator";
+import {playerStatsHasData} from "../../../../../Common/utils";
+import {abbreviateNumber} from "../../../common/utils";
+
+interface PingOverviewProps {
+    playerStats: PlayerStatModel[]
+}
+
+export const pings: Record<string, BaseStatGenerator> = {
```
Comment (afieldi) [inline front-end/src/components/player-page/stats/ping-overview.tsx:15]: Don't think this is required to be exported

File: front-end/src/components/player-page/stats/ping-overview.tsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+            <Box sx={{width: "30px"}}>
+                <Typography  fontFamily="Montserrat" align="center"
+                             variant="button">{abbreviateNumber(statGenerator.getStatNumber(playerStats))}</Typography>
+            </Box>
+
+        </Box>
+    );
+}
+
+function getImg(src: string) {
```
Comment (afieldi) [inline front-end/src/components/player-page/stats/ping-overview.tsx:80]: lol. big function

File: Common/utils.ts [hunk 3]
```diff
@@ -235,4 +235,8 @@ export function combine(object1: PlayerStatModel, object2: PlayerStatModel): Pla
 
 export function sigmoid(z: number, k: number) {
   return 1 / (1 + Math.exp(-z/k));
+}
+
+export function playerStatsHasData(playerStats: PlayerStatModel[]): boolean {
```
Comment (afieldi) [inline Common/utils.ts:240]: nit?: could this be replaced with just !!playerStats.length

I have no clue what this meant when I was reading.

File: front-end/src/common/stats-generators/pings/TotalEnemyMissingPingStatGenerator.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import {BaseStatGenerator} from "../BaseStatsGenerator";
+import PlayerStatModel from "../../../../../Common/models/playerstat.model";
+
+export class TotalEnemyMissingPingStatGenerator extends BaseStatGenerator {
+    getStatTitle(): string {
+        return "TEMP"
```
Comment (afieldi) [inline front-end/src/common/stats-generators/pings/TotalEnemyMissingPingStatGenerator.ts:6]: I know this stands for Total Enemy Missing Pings, but when I first saw this I did a double take

File: front-end/src/components/player-page/stats/ping-overview.tsx (hunk 1 above)
Comment (Vexrax) [reply front-end/src/components/player-page/stats/ping-overview.tsx:15]: Fixed in rev2

File: front-end/src/components/player-page/stats/ping-overview.tsx (hunk 2 above)
Comment (Vexrax) [reply front-end/src/components/player-page/stats/ping-overview.tsx:80]: Moved in rev2

File: Common/utils.ts (hunk 3 above)
Comment (Vexrax) [reply Common/utils.ts:240]: Changed name + function

**Review by afieldi — APPROVED** [review]

File: front-end/src/components/player-page/stats/ping-overview.tsx (hunk 1 above)
Comment (afieldi) [reply front-end/src/components/player-page/stats/ping-overview.tsx]: Tf is a rev? I see no revs on this site

**Pull request merged**